Lib/defcon/objects/features.py

```diff
@@ -66,4 +75,16 @@ def _get_text(self):
 
     def endSelfNotificationObservation(self):
         super(Features, self).endSelfNotificationObservation()
-        self._font = None
\ No newline at end of file
+        self._font = None
+
+    # -----------------------------
+    # Serialization/Deserialization
+    # -----------------------------
+
+    def getDataForSerialization(self, **kwargs):
+        getters = [('text', lambda k: getattr(self, k))]
```

Comment:
elsewhere `partial` is used for this instead of `lambda`. they both work; I just note this for consistency.